components/Documentation.tsx

```diff
@@ -32,6 +32,13 @@ export default function Documentation() {
           </ButtonV2Link>
         </FullWidthButtons>
 
+        <FullWidthButtons>
+          <span>If you need help with me, drop a message to my team:</span>
+          <ButtonV2Link href='https://filecoinproject.slack.com/archives/C017HM9BJ8Z'>
+            Slack Channel
```

Comment:
Hey @mur-me I'm going to attempt to set up a bridge from our discord -> slack so we have one "place" for Glif help